src/main/java/net/snowflake/client/log/SLF4JJJCLWrapperIT.java

```diff
@@ -0,0 +1,137 @@
+package net.snowflake.client.log;
```

Comment:
Same as for JDK14Logger- add new test class for SLF4JJCLWrapper class to run through functions